src/core/mod.rs

```diff
@@ -85,11 +95,19 @@ pub const BITCOIN_MAINNET_FIRST_BLOCK_TIMESTAMP: u32 = 1610643248;
 pub const BITCOIN_MAINNET_FIRST_BLOCK_HASH: &str =
     "0000000000000000000ab248c8e35c574514d052a83dbc12669e19bc43df486e";
 pub const BITCOIN_MAINNET_INITIAL_REWARD_START_BLOCK: u64 = 651389;
+pub const BITCOIN_MAINNET_STACKS_2_05_BURN_HEIGHT: u64 = 713_000;
+
+// TODO: Pick a real height for Stacks 2.1.
+pub const BITCOIN_MAINNET_STACKS_21_BURN_HEIGHT: u64 = 2_000_000;
```

Comment:
Can you just make this `u64::MAX`?  Same for `BITCOIN_TESTNET_STACKS_2_05_BURN_HEIGHT`